### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by zliebersbach**

### CODE REVIEW

**Comment by masonlouchart** [comment]
Maybe the documentation duplicated in the `readme` file must be removed. I tried to re-generate and publish the component page earlier but it fails. I don't know if the [tool](https://github.com/Polymer/tools/blob/master/bin/gp.sh) is till working...

**Comment by zliebersbach** [comment]
@LM450N The README should just link to the component page IMO. Also noticed some of the docs are duplicated in demo/index.html. That should probably be removed since the same docs can be seen in the "DOCS" page of the component page.

**Comment by zliebersbach** [comment]
@LM450N Fixed everything I mentioned above.

File: README.md [hunk 1]
```diff
@@ -1,116 +1,46 @@
...
-See the [Component Page][component_page]
+Check out the [component page][component_page].
 
 ## Usage
 
 1. Install The Component:
 
   ```sh
-  bower install toast-er --save
+  bower i -S toast-er
```
Comment (masonlouchart) [inline README.md]: Beginners do not know the shortcuts. I think it's preferable to leave the command with explicit arguments.

**Comment by masonlouchart** [comment]
I would like to solve the issue with the component page generator before merge this pull request. Simply because now the page is empty and the `README` is the only documentation available. And we have a review in progress.

**Comment by zliebersbach** [comment]
@LM450N I'll send a PR for the component page as soon as this is merged, so the updated docs will be in it. I don't think it'll be an issue if there are no working docs for a maximum of 24 hours (the maximum time it'll take me to get around to it).

**Comment by masonlouchart** [comment]
@wincinderith The component page is not broken, it's the generator of GitHub pages dedicated to Polymer elements. I don't know if we can fix it here because the "not found dependency" (in the console) is in the directory deployed on gitHub pages.

**Comment by zliebersbach** [comment]
I can fix it with a PR, trust me.

**Review by masonlouchart — CHANGES_REQUESTED** [review]
The command line to install the element must stay `bower install toast-er --save` because it's more explicit and understandable. People use shortcuts if they want.

**Comment by zliebersbach** [comment]
@LM450N Fixed!

**Comment by masonlouchart** [comment]
LGTM

**Review by masonlouchart — APPROVED** [review]

**Comment by zliebersbach** [comment]
@LM450N I have a working [`gh-pages`](https://github.com/wincinderith/toast-er/tree/gh-pages) branch here. You'll need to pull this branch locally and do a forced push to your version of the branch, as the branches have a different commit history.

**Comment by masonlouchart** [comment]
@wincinderith How have you generated this? I'm interesting to know to be able to do it myself for next releases.

**Comment by zliebersbach** [comment]
@LM450N Basically as follows in an empty directory on Unix:

```
git clone https://github.com/Polymer/tools
mkdir tmp && cd tmp
../tools/bin/gp.sh LM450N toast-er
```

You just need to make sure you have an SSH key setup on your system and
added to your GitHub account.

**Comment by masonlouchart** [comment]
@wincinderith I'm using the same tool but the result is not working 😞 ...

I think it's because I have a global `.gitignore` file which contains "/lib". I'm an idiot 👊 !

**Pull request merged**